src/utils/session-core.js

```diff
@@ -697,10 +697,10 @@ export const sendMessageTTL = async (session, messageObject, otherPodUsername) =
 
   const newTtlFile = buildThing(createThing({ name: 'document' }))
     .addDatetime('https://schema.org/uploadDate', date)
```

Comment:
Think we got an RDF_PREDICATE for uploadDate, `RDF_PREDICATE.uploadDate`, we're using it in Line 288 for the createDriversLicenseTtlFile function

Reply:
Yes, is this not correct? I included the base meta data that is part of the generic createResourceTtlFile() function as well just to have the name, endDate, serialNumber, description, and url as well.

Reply:
No, this is the correct url for that schema. It's just that since we're using RDF_PREDICATES for our schema, I was thinking we could just use that.

Reply:
Ah yes, it appears I missed that one when making the changes to that section of the code. Thanks for catching that!

Reply:
I'll be heading to bed in a bit. Nice to see another night owl like myself. If you think this is ready, feel free to pull it in, but if so, I'll make a separate branch/PR for documentation. Otherwise I'll simply start working on the docs on this branch. This will start either tomorrow or on Friday depending on how my other work is going. :peace: And again, thanks for everything!